### ISSUE TITLE
neo-python/examples/smart-contract.py cannot connect to peers on branch master

### ISSUE THREAD

**Issue by tomascarvalho** [CityOfZion/neo-python#593]
### Current behavior
- Using neo-python 0.7.7 installed through pip or neo-python cloned from master
- Unable to connect to peers and update blocks
- [Print Screen (Pip)](https://imgur.com/a/H4h3AAv) -> not working
- [Print Screen (master)](https://imgur.com/a/SlShktH) -> not working
- [Print Screen (development)](https://imgur.com/jNJB7bE) -> working

### Expected behavior
- Should connect to the testnetwork and listen to a smart-contract event

### How to reproduce

- Create and start a new virtualenv
- Install neo-python 0.7.7 through pip
- Copy neo-python/examples/smart-contract.py and run it inside the virtualenv

OR

- Clone and install neo-python from master
- Run neo-python/examples/smart-contract.py 

### Your environment

- OS: macOS High Sierra 10.13.6
- neo-python v0.7.7
- Python version: Python 3.6.4

*Note:* Is working fine when running this with development branch

**Comment by ixje** [comment]
I can confirm that from pip it indeed does not seem to connect to any nodes. Not sure why (yet)

**Comment by ixje** [comment]
Ok it's the seedlist that changed, that's why it's not working
```
/Users/erik/Documents/code/neo-python/venv/bin/python ./examples/smart-contract.py
[I 180904 19:11:07 LevelDBBlockchain:115] Created Blockchain DB at /Users/erik/.neopython/Chains/SC234 
[I 180904 19:11:09 smart-contract:55] Block 106847 / 405999
[I 180904 19:11:09 smart-contract:79] Everything setup and running. Waiting for events...
[I 180904 19:11:24 smart-contract:55] Block 109047 / 423999
```
If I update the seedlist then it starts adding blocks as seen above

**Comment by ixje** [comment]
sure! I'm using a virtual environment, so to find `protocol.testnet.json` holding the seedlist I have look into the installed packages list. For example, min can be found at
```
./venv/lib/python3.6/site-packages/neo/data/protocol.testnet.json
```

**Comment by localhuman** [comment]
Should now be fixed in `v0.7.8`
